### DISCUSSION TITLE
Disable "Kinetic Scrolling" effect ?

### DISCUSSION THREAD

**Discussion by Twiggeh** [hyprwm/Hyprland#1644] [Q&A]
I am sorry if this is the wrong place to ask, but it appears that after updating I have a form of "kinetic" scrolling effect enabled in any scrollable window, I am not certain this was caused by Hypr, but it was my best guess. Any ideas?

https://user-images.githubusercontent.com/23149166/221438471-e5c54b8b-6743-4d6c-917b-4f4960992989.mp4

**Comment by vaxerski** [comment]
I think it's a browser setting, no? @fufexan

**Reply by fufexan** [reply]
Yes, it's up to the application/toolkit to implement kinetic scroll.

**Reply by Twiggeh** [reply]
unfortunately they started detecting it oh well :( would be interesting ti figure out what the critera is for it to switch to kinetic mode
